ui/mods/msu/mod_settings/boolean_setting.js

```diff
@@ -1,12 +1,15 @@
 var BooleanSetting = function (_mod, _page, _setting, _parentDiv)
 {
 	this.data = _setting;
-	this.layout = $('<div class="boolean-container"/>');
+	this.layout = $('<div class="setting-container boolean-container"/>');
 	_parentDiv.append(this.layout);
-	this.checkbox = $('<input type="checkbox" id= "' + _mod.id + _setting.id + '-id" name="' + _setting.id +'-name" />');
-	this.layout.append(this.checkbox);
-	this.label = $('<label class="text-font-normal font-color-subtitle" for="' + _mod.id + _setting.id + '-id">' + _setting.name + '</label>');
-	this.layout.append(this.label);
+
+	this.titleContainer = $('<div class="setting-title-container"/>');
+	this.layout.append(this.titleContainer);
+
+	var id = _mod.id + _setting.id + "-id"
+	this.checkbox = $('<input type="checkbox" id= "' + id + '" name="' + _setting.id +'-name" />');
```

Comment:
Any reason this has been split up into 2 lines?

Reply:
It's cleaner, I would use format if I could, patching strings together with + concat is very messy

Reply:
If you say so